validate.sh

```diff
@@ -11,7 +11,7 @@ cargo clean -p mirai
 # Run format checks
 cargo fmt --all
 # Run lint checks
-cargo audit
+#cargo audit
```

Comment:
Why disable `cargo audit`?

Reply:
There is a security alert that we cannot fix locally.